src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+      <div class="item">
+        
+
+        <p class="item__code">Код товара: 195434</p>
+
+        <div class="review">
+          <block class="review__stars">
```

Comment:
```suggestion
          <block class="review__stars stars stars--4">
```
reuse code from `layout_stars` task with modifiers